src/HttpHandler/Logging.fs

```diff
@@ -15,6 +15,18 @@ module Logging =
     let private reqex =
         Regex(@"\{(.+?)(\[(.+?)\])?\}", RegexOptions.Multiline ||| RegexOptions.Compiled)
 
+    let private loggerFormatRegex =
+        Regex($@"{{{PlaceHolder.ResponseHeader}\[(.+?)\]}}", RegexOptions.Multiline ||| RegexOptions.Compiled)
+
+    let mutable private placeholderCounter = 0
+
+    let private incrementAndReturn (index: byref<int>) : int =
+        index <- index + 1
+        index
+
+    let private replacer (_: Match) : string =
+        $"{{{PlaceHolder.ResponseHeader}__{incrementAndReturn (&placeholderCounter)}}}"
```

Comment:
Could we instead use `Interlocked.Increment` from `System.Threading` here?

Reply:
Sure https://github.com/cognitedata/oryx/pull/137/commits/3b97fb55045d4cc1306fafa6830a65a9ddc5b7c2